src/scripts/webpack/webpack.build.js

```diff
@@ -12,7 +12,7 @@ export default ({ config, paths, NODE_ENV, ...options }) => {
     // cause failed production builds to fail faster
     new webpack.NoEmitOnErrorsPlugin(),
     new webpack.optimize.ModuleConcatenationPlugin(),
-    new ExtractTextPlugin({ filename: 'style.[hash].min.css' }),
+    new MiniCssExtractPlugin(),
```

Comment:
I think `filename` option can still be passed correctly (not sure if this is the default anyway..)